ops/terraform/services/server/modules/bfd_server_asg/main.tf

```diff
@@ -5,11 +5,38 @@ locals {
   rds_reader_endpoint = data.external.rds.result["CustomEndpoint"] == "" ? data.external.rds.result["ReaderEndpoint"] : data.external.rds.result["CustomEndpoint"]
 
   additional_tags = { Layer = var.layer, role = var.role }
-  scaleout_asg_capacities = [
-    { capacity = length(var.env_config.azs) * 2, metric_lower_bound = 1 * var.scaling_networkin_interval_mb, metric_upper_bound = 2 * var.scaling_networkin_interval_mb },
-    { capacity = length(var.env_config.azs) * 3, metric_lower_bound = 2 * var.scaling_networkin_interval_mb, metric_upper_bound = 4 * var.scaling_networkin_interval_mb },
-    { capacity = length(var.env_config.azs) * 4, metric_lower_bound = 4 * var.scaling_networkin_interval_mb, metric_upper_bound = null }
+
+  # FUTURE: Encode the scaling step, scaling stages, and alarm evaluation periods within config
+  scaling_capacity_step = length(var.env_config.azs)
+  scaling_alarms_config = {
+    # Scale-out alarm is configured to ALARM if 1 out of 1 consecutive 1 minute periods report high
+    # network traffic. This is intended to scale the Server aggressively
+    scale_out = {
+      eval_period                  = 1 * 60
+      datapoints_to_alarm          = 1
+      consecutive_periods_to_alarm = 1
+    }
+    # Scale-in alarms are configured to ALARM if 15 out of 15 consecutive 1 minute periods report
+    # low network traffic. This is intended to keep the Server scaled-out as traffic is often bursty
+    # over a given period rather than sustained at a given threshold
+    scale_in = {
+      eval_period                  = 1 * 60
+      datapoints_to_alarm          = 15
+      consecutive_periods_to_alarm = 15
+    }
+  }
+  scaling_stages = [
+    # Between 0 MB/min and 100 MB/min we want to be at 3 instances
```

Comment:
Huge nitpick and not important, but I would have these comments say something like "between 0 MB/min and 1x the scaling interval MB/min..." so we don't have to update these comments/have misleading comments should that interval change w/o updating the description

Reply:
also - one of the first "gotcha" thoughts I had was to make understand which values between `begin` and `end` were exclusive/inclusive - can we make a comment for that here?

Reply:
I agree. I'll reword these comments to indicate the which bounds are inclusive/exclusive. I'll also make the values more generic.